src/RA_LeaderboardPopup.cpp

```diff
@@ -263,8 +266,8 @@ _Use_decl_annotations_ void LeaderboardPopup::Render(ra::ui::drawing::ISurface&
 
                         const auto sScore = ra::Widen(pLB->FormatScore(lbInfo.m_nScore));
                         const auto szScore = m_pScoreboardSurface->MeasureText(nFontText, sScore);
-                        m_pScoreboardSurface->WriteText(m_pScoreboardSurface->GetWidth() - 4 - szScore.Width - 8, nY,
-                                                        nFontText, nTextColor, sScore);
+                        const auto nX = gsl::narrow_cast<int>(m_pScoreboardSurface->GetWidth()) - 4 - szScore.Width - 8;
```

Comment:
this should use `iWidth` too.

Reply:
Sure, I guess `iWidth` should be `nWidth` as well.